content/en/real_user_monitoring/browser/setup/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: RUM Browser Monitoring Setup
+title: Browser Monitoring Setup
```

Comment:
Just noticed that in some place like [here](https://github.com/DataDog/documentation/blob/a7e0f18152a23b60da2b9e8fe5869a20d5d74088/content/en/real_user_monitoring/browser/setup/server.md) it's still `RUM Browser Monitoring`.

Reply:
ok thank you I'll get that fixed